File: JAVA/Calculator.java
import java.util.*;

public class Calculator {
    public static void main(String args[]) {
        Scanner sc = new Scanner (System.in);
        
        System.out.print("Enter first number: ");
        double num1 = sc.nextInt();

        System.out.print("Enter the operator (+,-,*,/,%): ");
        char op = sc.next().charAt(0);

        System.out.print("Enter second number: ");
        double num2 = sc.nextInt();

        switch (op) {
            case '+':
                System.out.println("Result: "+(num1 + num2));
                break;

            case '-':
                System.out.println("Result: "+(num1 - num2));
                break;

            case '*':
                System.out.println("Result: "+(num1 * num2));
                break;

            case '/':
                if (num2 != 0) {
                    System.out.println("Result: "+(num1 / num2));
                    break;    
                } else { 
                    System.out.println("Error: Cannot divide by zero!");
                    break; }

            case '%':
                if (num2 != 0) {
                    System.out.println("Result: "+(num1 % num2));
                    break;    
                } else { 
                    System.out.println("Error: Cannot divide by zero!");
                    break; }
                    
            default:
                System.out.println("Invalid operator");
                break;
        }

        sc.close();
    }
}

File: C/Calculator.c
#include <stdio.h>
#include <stdbool.h>

int main()
{
    float num1, num2, ans;
    char operator;
    bool status = true;

    printf("Enter the operator (+, -, *, /): ");
    scanf("%c",&operator);

    printf("Enter first value : ");
    scanf("%f",&num1);

    printf("Enter second value : ");
    scanf("%f",&num2);

    switch (operator)
    {
    case '+' :
        ans = num1 + num2;
        break;
    
    case '-' :
        ans = num1 - num2;
        break;

    case '*' :
        ans = num1 * num2;
        break;

    case '/' :
        if(num2 != 0){
            ans = num1 / num2;
        } else {
            printf("Error: Cannot divide by zero!");
        }
        break;
    
    default:
        printf("Error: Invalid operator!");
        status = false;
        break;
    }

    if(status){
        printf("result : %.2f %c %.2f = %.2f",num1, operator, num2, ans);
    }

    return 0;
}